src/Core/Components/Dialog/Parameters/DialogParameters.cs

```diff
@@ -20,10 +20,16 @@ public class DialogParameters : ComponentParameters, IDialogParameters
 
     /// <summary>
     /// Determines if the dialog is modal. Defaults to true.
-    /// When true, clicking outside the dialog will dismiss the dialog.
+    /// Partially obscures the area around the dialog.
     /// </summary>
     public bool? Modal { get; set; } = true;
 
+    /// <summary>
+    /// Determines if a modal dialog is dismissible by clicking
+    /// outside the dialog. Defaults to false.
+    /// </summary>
+    public bool? InertModalOverlay { get; set; } = false;
```

Comment:
Rename to `PreventDismissOnOverlayClick` please to make it more clear what it is and does.